sample-data/src/test/java/com/kodemon/sampledata/SampleDataLoadingFacadeTest.java

```diff
@@ -0,0 +1,45 @@
+package com.kodemon.sampledata;
+
+import com.kodemon.persistence.dao.GymDao;
+import com.kodemon.persistence.dao.PokemonDao;
+import com.kodemon.persistence.dao.TrainerDao;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.springframework.transaction.annotation.Transactional;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.persistence.EntityManager;
```

Comment:
Right-click project root in IDEA --> Optimize imports for the whole project :smile: